src/components/Dropdown/index.js

```diff
@@ -191,4 +195,13 @@ Dropdown.prototype = Object.assign(Dropdown.prototype, Component.prototype, {
       placeholder: this.getPlaceholder(),
     };
   },
+
+  getID() {
+    return this.selected.get('dropdown-toggle').id;
+  },
+
+  setID(id = '') {
+    this.selected.get('dropdown-toggle').id = id;
+    this.emit('id:changed', id);
+  },
```

Comment:
Aqui mesma coisa

Reply:
Esse e nos casos iguais que vc comentou eu tinha colocado o ID só pra caso a pessoa que vá fazer o sistema de login precisasse dos IDs nos campos do form, mas se for desnecessário eu posso remover

Reply:
Entendi, então ta, por mim precisa remover não